lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -16,16 +16,10 @@ class CheckoutV2Gateway < Gateway
       LIVE_ACCESS_TOKEN_URL = 'https://access.checkout.com/connect/token'
       TEST_ACCESS_TOKEN_URL = 'https://access.sandbox.checkout.com/connect/token'
 
-      def initialize(options = {})
-        @options = options
-        @access_token = options[:access_token] || nil
+      BASIC_SECRET_KEY_FORMAT = /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i
 
-        if options.has_key?(:secret_key)
-          requires!(options, :secret_key)
-        else
-          requires!(options, :client_id, :client_secret)
-          @access_token ||= setup_access_token
-        end
+      def initialize(options = {})
```

Comment:
I think you might have accidentally added all the work from the other PR.

Reply:
Yes, I already updated this PR